drakrun/drakrun/storage.py

```diff
@@ -10,6 +8,13 @@
 
 from typing import Generator, Tuple
 from drakrun.config import InstallInfo, VOLUME_DIR
+from drakrun.util import safe_delete
+
+logging.basicConfig(
+    level=logging.DEBUG,
+    format='[%(asctime)s][%(levelname)s] %(message)s',
+    handlers=[logging.StreamHandler()]
+)
```

Comment:
This is a library module. Importing it will trigger the `basicConfig`. This shouldn't be here

Reply:
Sure, will remove it